cmake/configure/configure_2_trilinos.cmake

```diff
@@ -213,6 +207,27 @@ MACRO(FEATURE_TRILINOS_FIND_EXTERNAL var)
         ADD_FLAGS(CMAKE_REQUIRED_FLAGS "--expt-extended-lambda")
       ENDIF()
 
+      # We need a recent version of Trilinos to use kokkos_check. We want to use
+      # VERSION_GREATER_EQUAL 13.0 but this requires CMake 3.7
+      IF(TRILINOS_VERSION VERSION_GREATER 12.99)
+        SET(KOKKOS_WITH_OPENMP "")
+        kokkos_check(DEVICES OpenMP RETURN_VALUE KOKKOS_WITH_OPENMP)
+        IF(KOKKOS_WITH_OPENMP)
+          FIND_PACKAGE(OpenMP REQUIRED)
+          ADD_FLAGS(DEAL_II_CXX_FLAGS "${OpenMP_CXX_FLAGS}")
+          ADD_FLAGS(DEAL_II_LINKER_FLAGS "${OpenMP_CXX_FLAGS}")
```

Comment:
Is this going to conflict with other places where we may have already configured OpenMP, or will still?

Reply:
This works on the configurations I have tried and this includes other packages that set CUDA. This only sets `-fopenmp`. That's all it does.